dashboard/app/views/pd/workshop_mailer/exit_survey.html.haml

```diff
@@ -0,0 +1,95 @@
+
+%p
+  Thank you for attending a Code.org
+  = @workshop.course
+  = @workshop.subject
+  workshop and for your commitment to bring foundational 21st century learning to your students!
+
+- if @is_first_workshop
+  %p
+    Computer science has become the fastest-growing subject in K-12 education,
+    thanks to inspirational educators like you who have decided to spend your summer or weekends
+    in professional learning for the benefit of your classroom and the future of our children.
+    Thank you for joining the teacher-powered movement to give every student in every school
+    the opportunity to learn computer science.
+
+%p
+  = link_to('Please take a minute to complete this short survey', @survey_url)
+  Let us know how we can improve your experience and your learning.
+  If you had a good experience, please spread the word about the
+  = link_to('Code.org Professional Learning Program', 'https://code.org/educate') + '.'
+
+= render partial: 'share_links'
+
+- if @workshop.course == Pd::Workshop::COURSE_CSF
+  %h3
+    Classroom supplies, at no cost:
+  %p
+    Completing the survey will qualify you to receive supplies at no cost for the unplugged activities.
+
+  %p
+    Attached to this email, you will also find a personalized certificate acknowledging your successful
+    completion of Code.org's CS Fundamentals workshop.
+
+  %h3
+    For a limited time
+  %p
+    Code.org is offering $50 off
+    = link_to('Dash robots', 'https://code.org/l/PCNgRYjZ-G4LZOyuAg5wXQ==/Mzc1ODY2') + '.'
+    These hands-on learning tools reinforce CS concepts taught in Code.org's CS Fundamentals curriculum,
+    like events and loops.
+    = link_to('Learn more', 'https://code.org/l/PCNgRYjZ-G4LZOyuAg5wXQ==/Mzc1ODY2') + '.'
+    %br
+    %i
+      Valid March 1st-31st, 2016 only. Limited to first 1,000 people.
+
+  %p
+    %strong
+      Redemption code for robots: CODE1389M
+    %br
+    %i
+      Valid March 1-31st, 2016 only. Limited to first 1,000
+    %br
+    = link_to 'Go here', 'https://code.org/l/PCNgRYjZ-G4LZOyuAg5wXQ==/MzUxNzIx'
+    for questions regarding redemption.
+
+%h3
+  Need more support?
+
+%ul
+  %li
+    Connect with other educators teaching CS. Join our
+    = link_to('Professional Learning Community', 'https://code.org/l/PCNgRYjZ-G4LZOyuAg5wXQ==/MjM4NjQy') + '.'
+  %li
+    See our
+    = link_to('FAQ', 'https://code.org/l/PCNgRYjZ-G4LZOyuAg5wXQ==/NDg4') + '.'
+  %li
+    Or
+    = link_to('contact us', 'https://code.org/l/PCNgRYjZ-G4LZOyuAg5wXQ==/NDg5') + '.'
+
+%p
+  If there’s anything else my team or I can help you with, send us an email.
+
+%p
+  Thanks again for all you do for our students,
+
+%p
+  Hadi Partovi
+  %br
+  Founder and CEO, Code.org
+
+%p
+  Follow us
+  = link_to 'on Facebook', 'http://facebook.com/code.org'
+  or
+  = link_to 'on Twitter', 'http://twitter.com/codeorg'
+  %br
+  Code.org is a public 501c3. Our address is 1301 5th Ave, Suite 1225, Seattle, WA, 98101.
```

Comment:
I don't think US addresses have commas after state and before ZIP code, do they?